core/domain/question_services.py

```diff
@@ -201,3 +233,73 @@ def update_question(committer_id, question_id, change_list, commit_message):
     """
     question = apply_change_list(question_id, change_list)
     _save_question(committer_id, question, change_list, commit_message)
+
+
+def add_question_id_to_skill(question_id, collection_id, skill_id, user_id):
+    """Adds the question id to the question list of the appropriate skill.
+
+    Args:
+        question_id: str. The id of the question.
+        collection_id: str. The id of the collection.
+        skill_id: str. The id of the skill.
+        user_id: str. The id of the user.
+    """
+    collection_services.update_collection(
+        user_id, collection_id, [{
+            'cmd': collection_domain.CMD_ADD_QUESTION_ID_TO_SKILL,
+            'skill_id': skill_id,
+            'question_id': question_id
+        }], 'Add a new question with ID %s to skill with ID %s'%(
+            question_id, skill_id))
+
+
+def remove_skill(question_id, collection_id, skill_id, user_id):
```

Comment:
Are you removing a question or a skill here?

Reply:
It removes question id from collection_skill object property question_ids.

Reply:
Hm, then shouldn't it be called remove_question_id_from_skill, or similar?

Reply:
Updated.